genai-perf/genai_perf/inputs/retrievers/generic_dataset.py

```diff
@@ -81,6 +80,6 @@ def to_dict(self) -> GenericDatasetDict:
         }
```

Comment:
is the example in the docstring still correct?

Reply:
Thanks for checking. This docstring is still correct. The output format wasn't changed.